META-INF/maven/org.openmrs.module/isanteplus-api/pom.xml

```diff
@@ -0,0 +1,92 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced